be/src/service/arrow_flight/flight_sql_statement_batch_reader.cpp

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "service/arrow_flight/flight_sql_statement_batch_reader.h"
+
+#include <arrow/status.h>
+
+#include "arrow/builder.h"
+#include "runtime/exec_env.h"
+#include "runtime/result_buffer_mgr.h"
+#include "util/arrow/row_batch.h"
+#include "util/arrow/utils.h"
+
+namespace doris {
+namespace flight {
+
+std::shared_ptr<arrow::Schema> DorisStatementBatchReader::schema() const {
+    return schema_;
+}
+
+DorisStatementBatchReader::DorisStatementBatchReader(std::shared_ptr<DorisStatement> statement,
+                                                     std::shared_ptr<arrow::Schema> schema)
+        : statement_(std::move(statement)), schema_(std::move(schema)) {}
+
+arrow::Result<std::shared_ptr<DorisStatementBatchReader>> DorisStatementBatchReader::Create(
```

Comment:
create...
using doris's code format , not use arrow code format

Reply:
ok, done